File: 벤치마크/java/Fibonacci.java
public class Fibonacci {
    static int fib(int n) {
        if (n <= 1) return n;
        return fib(n - 1) + fib(n - 2);
    }

    public static void main(String[] args) {
        long start = System.nanoTime();
        int result = fib(40);
        double ms = (System.nanoTime() - start) / 1e6;
        System.out.println("fib(40) = " + result);
        System.out.printf("RESULT:%.3f%n", ms);
    }
}

File: 벤치마크/go/fibonacci.go
package main

import (
	"fmt"
	"time"
)

func fib(n int) int {
	if n <= 1 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	start := time.Now()
	result := fib(40)
	elapsed := time.Since(start).Seconds() * 1000
	fmt.Printf("fib(40) = %d\n", result)
	fmt.Printf("RESULT:%.3f\n", elapsed)
}
